support/integration-testing/docker-compose.bruno.yml

```diff
@@ -27,7 +27,7 @@ services:
       --output /opt/redbox-portal/.tmp/junit/backend-bruno/backend-bruno.xml 
       --bail"
   redboxportal:
-    image: qcifengineering/redbox-portal:develop
+    image: qcifengineering/redbox-portal:feature-node24
```

Comment:
The `bruno` service in this file uses the `node` image, should that be updated too?